sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.3 (Unreleased)
 
+- Breaking change: Disabled `UsernamePasswordCredential` in browser bundles. This credential was never supported in browsers, but now that it uses MSAL, having it in browsers would break the browser bundles.
```

Comment:
What was the exact behavior in v1 when someone used UsernamePasswordCredential in the browser?

Reply:
It would work if CORS was disabled too: https://github.com/Azure/azure-sdk-for-js/blob/hotfix/identity_1.3.0/sdk/identity/identity/src/credentials/usernamePasswordCredential.ts should I bring this credential back as well?

Reply:
If we are making ClientSecretCredential behavior in browser to be the same as in v1, shouldnt the same be applied to other credentials (other than of course InteractiveBrowserCredential)?

Reply:
We can touch the other credentials in a different PR, but since UsernamePasswordCredential is anyway being touched here, we should consider making the browser experience to be same as v1

Reply:
I can add `UsernamePasswordCredential` too. I feel like we should drop the one s we're not using to test stuff, but adding it would align us a little better with v1, so I'll add it back.

Reply:
Pushed the v1 version of UsernamePasswordCredential on browsers 👍